docs/guide/content_management.md

```diff
@@ -55,7 +57,8 @@ A User is simply a Content item of the User account Content Type.
 The Users are organized within User Group Content items below this Location.
 In other words, the **Users** Location contains the actual Users and User Groups,
 which can be viewed by selecting the **Users** tab in the Admin Panel.
-The default ID number of the **Users** Location is 5; it contains a User Group Content item.
+The default ID number of the **Users** Location is 5, it can be [modified via configuration](config_repository.md#top-level-locations).
```

Comment:
```suggestion
The default ID number of the **Users** Location is 5, but it can be [modified via configuration](config_repository.md#top-level-locations).
```